modules/020-deckhouse/docs/USAGE.md

```diff
@@ -75,3 +75,17 @@ Manual confirmation of the update to the version `v1.26.0`:
 ```shell
 kubectl patch DeckhouseRelease v1-26-0 --type=merge -p='{"approved": true}'
 ```
+
+### Collect debug info
+
+We always appreciate helping users with debugging complex issues. To collect all necessary info, a user should:
+
+1. Execute the following command:
+
+```sh
+kubectl -n d8-system exec deploy/deckhouse \
+  -- deckhouse-controller helper collect-debug-info \
+  > deckhouse-debug-$(date +"%Y_%m_%d").tar.gz
+```
+
+2. Send the archive to the Deckhouse team for further debugging.
```

Comment:
Maybe it's a good idea to provide some contacts where users could send this archive, a telegram channel link for example.

Reply:
The only option I can suggest is https://deckhouse.io/en/community/about.html#online-community

Reply:
@nabokihms Is it more appropriate to use https://deckhouse.io/en/community/about.html#online-community instead of a new issue link? What do you think?